src/types/CoverageEntry.ts

```diff
@@ -29,6 +29,7 @@ export interface CoverageEntry {
     note_content_text: string;
     attachment: UploadedFile | UploadedImage | null;
     attachment_oembed: OEmbedInfo | null;
+    screenshot: UploadedFile | null;
```

Comment:
What's this property for? Should it be `UploadedImage`?

Reply:
Right, right, right! Thanks!

Reply:
https://github.com/prezly/javascript-sdk/pull/310/commits/69895beab705515295ee28777336ad984862d229